zipline/algorithm.py

```diff
@@ -495,8 +507,21 @@ def run(self, source, overwrite_sim_params=True,
             # if DataFrame provided, map columns to sids and wrap
             # in DataFrameSource
             copy_frame = source.copy()
+
+            # Build new Assets for identifiers that can't be resolved as
```

Comment:
DataFrameSource and PanelSource objects now only build Assets that are not able to be mapped. This is primarily for test compatibility.